source/minikube.rst

```diff
@@ -0,0 +1,80 @@
+Install Percona Server for MongoDB on minikube
+==============================================
+
+Installing PSMDB Operator on `minikube <https://github.com/kubernetes/minikube>`_
+is the easiest way to try it locally without a cloud provider. Minikube runs
+Kubernetes on GNU/Linux, Windows, or macOS system using a system-wide
+hypervisor, such as VirtualBox, KVM/QEMU, VMware Fusion or Hyper-V. Using it is
+a popular way to test Kubernetes application locally prior to deploying it on a
+cloud.
+
+Following steps are needed to run PSMDB Operator on minikube:
+
+0. `Install minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_, using a way recommended for your system. This includes the installation of the following three components:
```

Comment:
Do we intentionally start numbering from 0 because this is a prerequisite we note or should we begin numbering from 1 as is typical in documentation?